packages/core/src/modules/ledger/IndyPool.ts

```diff
@@ -20,6 +23,7 @@ export interface IndyPoolConfig {
   genesisTransactions?: string
   id: string
   isProduction: boolean
+  didIndyNamespace: DidIndyNamespace
```

Comment:
Why do we use `didIndyNamespace` in some places and `indyNamespace` in others? I think it would be nice to stay consistent across the codebase